src/toolbars/toolbar-component.js

```diff
@@ -1,6 +1,7 @@
 angular.module('patternfly.toolbars').component('pfToolbar', {
   bindings: {
-    config: '='
+    config: '=',
+    onToggleColvis: '&'
```

Comment:
colvis is not intuitive as to what it is used for. Please spell it out as onToggleColumnVisibility